pkg/controllers/managementuserlegacy/systemimage/systemimage.go

```diff
@@ -119,7 +118,5 @@ func GetSystemImageVersion() (string, error) {
 }
 
 func getSystemService() map[string]SystemService {
-	return map[string]SystemService{
-		alerting.ServiceName: alerting.NewService(),
-	}
+	return map[string]SystemService{}
```

Comment:
Returning empty service is making the whole systemimage package redundent. So, should we remove the entire package ? 

Returning empty service here makes  GetSystemImageVersion func useless but this function is used in Auth package. Don't know if this part of codebase is still relevant ?

Reply:
Seems like it can be removed after reading the original PR description https://github.com/rancher/rancher/pull/35564

The description says only start controller if alert, logging and pipeline controllers are started which are all removed afaik

Reply:
Also, the removal logging v1  https://github.com/rancher/rancher/pull/39189/files#diff-04d201a0e0a8a07d822db5be4cdce9a88a7533eec1da6a863758d6a71eaefd77 also does the same thing